packages/react-formio/src/components/table/hooks/useOperations.hook.tsx

```
@@ -1,11 +1,11 @@
-import React from "react";
+import React, { type ComponentType, type FunctionComponent } from "react";
 import { Hooks } from "react-table";
 
 import { Operation } from "../../../interfaces";
 import { DefaultCellOperations } from "../components/defaultCellOperations.component";
 
 export type UseOperationsHookProps = {
-  CellOperations?: React.ComponentType;
+  CellOperations?: FunctionComponent | ComponentType<{}>;
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)